checks/f5_bigip_fans

```diff
@@ -41,9 +41,12 @@ def parse_f5_bigip_fans(info):
             if fanchoice >= len(fantyp):
                 continue
             if fanchoice == 0:
-                parsed[("%s %d" % (fantyp[fanchoice], int(fanentry[0])))] = int(fanentry[1])
+                parsed[("%s %d" % (fantyp[fanchoice], int(fanentry[0])))] = (
+                    int(fanentry[2]),
+                    int(fanentry[1])
```

Comment:
Is it guaranteed that the fan status OID is also present in older models?

Reply:
The OID is part of the official MIB and even located in between the index and fan speed. 
There should no case that is missing. Also, it's only considered if the fan speed is zero.

Reply:
I found the OID in official MIB. I wanted to make sure because the OID would be an empty string if it is missing and that causes the int conversion to throw an exception. Than the check would fail when parsing and stop working.

Reply:
I removed the typecast and compare with String now. So if value is missing, it would be None and None == "1" will not hit our cause a Problem.